nodelet/pkg/pf9kube/pf9/pf9-kube/conf/masterconfig/base/centos/master.yaml

```diff
@@ -12,6 +12,8 @@ spec:
       command:
         - "kube-controller-manager"
         - "--cloud-provider=__CLOUD_PROVIDER__"
+        - "--cluster-signing-cert-file=/srv/kubernetes/certs/apiserver/request.crt"
```

Comment:
Instead of modifying the YAML directly can you use `CONTROLLER_MANAGER_FLAGS` to set these flags?